### ISSUE TITLE
[cpp.predefined] Use of "an" broads condition too much

### ISSUE THREAD

**Issue by JohelEGP** [cplusplus/draft#5715]
Originating comment: https://github.com/cplusplus/draft/pull/5670#discussion_r932606422.

> - __STDCPP_BFLOAT16_T__
Defined as the integer literal 1 if and only if the implementation supports an extended floating-point type with the properties described in [basic.extended.fp].

The use of "an" suggests that if any other "extended floating-point type with the properties described in [basic.extended.fp]" other than `std::float16_t` is defined by the implementation, then `__STDCPP_BFLOAT16_T__` should be defined to 1 even if `std::float16_t` isn't.

**Comment by dkolsen-pgi** [comment]
You are right.  I'm not sure how I messed that one up.  There must have been some context that got lost when the wording was moved to [cpp.predefined].

I suggest:

> Defined as the integer literal 1 if and only if the implementation supports an extended floating-point type with the properties of the _typedef-name_ `std::bfloat16_t` as described in [basic.extended.fp].

**Comment by tkoeppe** [comment]
@jensmaurer Do you consider fixing that editorial or would you prefer CWG/EWG oversight?

**Comment by jensmaurer** [comment]
This looks editorial to me.
